listenbrainz/listenstore/tests/test_timescale_utils.py

```diff
@@ -0,0 +1,107 @@
+import logging
+
+from brainzutils import cache
+from sqlalchemy import text
+
+import listenbrainz.db.user as db_user
+from listenbrainz.db import timescale
+
+from listenbrainz.db.testing import DatabaseTestCase, TimescaleTestCase
+from listenbrainz.listenstore import TimescaleListenStore
+from listenbrainz.listenstore.tests.util import create_test_data_for_timescalelistenstore
+from listenbrainz.listenstore.timescale_utils import recalculate_all_user_data, update_user_listen_data, \
+    delete_listens
+from listenbrainz.webserver import create_app
+
+
+class TestTimescaleUtils(DatabaseTestCase, TimescaleTestCase):
+
+    def setUp(self):
+        DatabaseTestCase.setUp(self)
+        TimescaleTestCase.setUp(self)
+        self.log = logging.getLogger(__name__)
+        self.app = create_app()
+        self.logstore = TimescaleListenStore(self.log)
+
+    def tearDown(self):
+        self.logstore = None
+        DatabaseTestCase.tearDown(self)
+        TimescaleTestCase.tearDown(self)
+        cache._r.flushdb()
+
+    def _create_test_data(self, user, file=None):
+        test_data = create_test_data_for_timescalelistenstore(user["musicbrainz_id"], user["id"], file)
+        self.logstore.insert(test_data)
+        return len(test_data)
+
+    def _get_count_and_timestamp(self, user):
+        with timescale.engine.connect() as connection:
+            result = connection.execute(
+                text("""
+                    SELECT count, min_listened_at, max_listened_at
+                      FROM listen_user_metadata
+                     WHERE user_id = :user_id
+                """), user_id=user["id"])
+            return dict(**result.fetchone())
+
+    def _get_pending_deletes(self):
```

Comment:
A get function that returns nothing and prints stuff. Odd?

Reply:
I see that you have some debugging stuff leftover here. Are you confident enough to clean that up now?